examples/search_examples/public/search/app.tsx

```diff
@@ -737,6 +785,15 @@ export const SearchExamplesApp = ({
                 selectedTab={reqTabs[selectedTab]}
                 onTabClick={(tab) => setSelectedTab(reqTabs.indexOf(tab))}
               />
+              <EuiSpacer />
+              {currentAbortController && (
```

Comment:
Should we hide this button when the search result completes as aborting doesn't make sense any more afterwards?

Reply:
Agree, had to add `isLoading` flag to track this